src/puppetlabs/puppetdb/http/query.clj

```diff
@@ -72,6 +118,18 @@
     req
     (restrict-query ["=" ["node" "active"] true] req)))
 
+(defn restrict-query-to-active-nodes'
```

Comment:
Unsure about this convention in the large. We could first change all the existing ones to -old or -dep, then introduce the new one with the regular name?